R-package/tests/testthat/test_dataset.R

```diff
@@ -404,6 +421,8 @@ test_that("lgb.Dataset: should be able to run lgb.cv() immediately after using l
     , metric = "binary_logloss"
     , num_leaves = 5L
     , learning_rate = 1.0
+    , num_iterations = 5L
```

Comment:
note for you @jmoralez : here's the one line in this PR not *strictly* related to logging. Adding `verbosity` below is sufficient to suppress all log lines from this test.

But while looking at it, I noticed that this was running for 100 iterations (because I saw all the logs before silencing them). Reducing that to 5 just to save CI time.